frontend/dockerfile/docs/reference.md

```diff
@@ -1462,6 +1463,41 @@ attempted to be used instead.
 > guide – Leverage build cache](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache)
 > for more information.
 
+### COPY --from
```

Comment:
Reminds me of https://github.com/moby/buildkit/issues/816. Maybe we should add a not about `COPY --from` being not supported.

Reply:
done, see c0b3994
(can we deprecate this instruction soon 😢 )